source/direct_debit/index.html.md.erb

```diff
@@ -5,248 +5,126 @@ weight: 80
-    <td> The issuing bank has confirmed that funding source (for example, Direct Debit mandate) is valid. </td>
-    </tr>
-    <tr>
-    <td nowrap><code> PENDING </code></td>
-    <td nowrap><code> FAILED </code></td>
-    <td> The issuing bank has rejected the funding source. Possible reasons: incorrect bank account details, bank account requires multiple signatories to set up a Direct Debit. </td>
-    </tr>
-    <tr>
-    <td nowrap><code> PENDING </code></td>
-    <td nowrap><code> CANCELLED </code></td>
-    <td> Cancelled by the payer. </td>
-    </tr>
-    <tr>
-    <td nowrap><code> ACTIVE </code></td>
-    <td nowrap><code> CANCELLED </code></td>
-    <td> Cancelled by the payer. </td>
-    </tr>
-  </table>
-</div>
+1. Select **Link GoCardless Merchant Account** and follow the on-screen instructions.
 
-## One-off payments using Direct Debit
+You must [contact us](/support_contact_and_more_information/#contact-us) to sign your GoCardless contract before you switch to live.
 
-To take a single payment using Direct Debit, you must set up a Direct Debit mandate.
+## Set up a mandate
 
-1. Create an API key linked to your Direct Debit service.
+You need to set up a Direct Debit 'mandate' before you can take payments. A mandate is your user's agreement that you can take payments from their bank account.
 
-1. Follow the instructions in the [“Payment flow” section](/payment_flow) of this documentation.
+These steps happen in order.
 
-The same API call is used for single payments in all contexts, including
-Direct Debit:
+### 1. GOV.UK Pay creates the mandate
 
-POST /v1/payments
+API call: `POST /v1/directdebit/mandates`
 
+Include the following parameters in your API call:
 
-You cannot take further payments from users using this method without creating
-a new payment. This requires users to re-enter their account details.
+- `description`
+- `reference`
+- `return_url`
 
-## Variable payments using Direct Debit
+#### description
 
-You can collect payments of variable amounts at any time by using a variable agreement.
-For example you may want to collect:
+`description` is a human-readable description of the mandate.
 
-* money each time a service is provided
+The description must be no longer than 255 characters, and must not contain URLs.
 
-* payments where the amounts vary according to service usage
+#### reference
 
-* regular payments where you do not know in advance how much each payment will be
+`reference` is the reference number you want to associate with this mandate.
 
-You could also use this functionality to simulate a recurring or
-scheduled Direct Debit payment, by sending an API call on a set schedule.
-Users would be emailed each time a payment is collected against their mandate.
+The reference number does not need to be unique, and must not:
 
-### Variable Direct Debit payment process
+- be longer than 255 characters
+- contain URLs
 
-These steps happen in order.
+The format is variable.
```

Comment:
This is borrowed from https://docs.payments.service.gov.uk/making_payments/#reference - it's about teams being able to use string/number/etc as they like, rather than being restricted to one format. Let me know if it doesn't make sense here though.

Reply:
I see. Maybe we could say something along the lines of, “You can use any format that’s suitable for your service as long as you follow the rules above.”